src/style.css

```diff
@@ -1,123 +1,212 @@
-* {
-    margin: 0;
-    padding: 0;
-    box-sizing: border-box;
-}
-body {
-    background-image: url('https://user-images.githubusercontent.com/55853256/91184100-daa3f700-e6b1-11ea-96b2-62bb82ed786e.jpg');
-    background-repeat: no-repeat;
-    background-size: cover;
-    background-position: center center;
-    background-attachment: fixed;
-}
-.container {
-    width: 100%;
-    /*el max width determina cuántas imágenes abarcar*/
-    max-width: 800px;
-    margin: auto;
-}
-/* HEADER */
-header {
-    /*dejar with de quitar el container*/
-    width: 100%;
-    height: 600px;
-}
-header nav {
-   height: 70px;
-   display: flex;
-   align-items: center;
-   justify-content: space-between;
-   background-image: url("images/nav_bkg.png");
-   /*background-size: contain;*/
-   background-position: center;
-}
-.hamb-menu {
-    padding: 16px;
-}
-nav .title {
-    margin-left: 72px;
-    color: #94E858;
-}
-header form {
-    width: 100%;
-    display: flex;
-    justify-content: center;
-}
-header .search-bar {
-    width: 70%;
-    height: 40px;
-* {
-    margin: 0;
-    padding: 0;
-    box-sizing: border-box;
-}
-body {
-    /*solo para mostrar contraste*/
-    background: lightseagreen;
-}
-.container {
-    width: 100%;
-    margin: auto;
-}
-/* HEADER */
-header {
-    /*dejar with de quitar el container*/
-    width: 100%;
-    height: 600px;
-}
-header nav {
-   height: 70px;
-   display: flex;
-   align-items: center;
-   justify-content: space-between;
-   background-image: url("images/nav_bkg.png");
-   /*background-size: contain;*/
-   background-position: center;
-}
-.hamb-menu {
-    padding: 16px;
-}
-nav .title {
-    margin-left: 72px;
-    color: #94E858;
-}
-header form {
-    width: 100%;
-    display: flex;
-    justify-content: center;
-}
-header .search-bar {
-    width: 70%;
-    height: 40px;
-    line-height: 40px;
-    background: white;
-    padding: 0 20px;
-    border-radius: 10px;
-    border: none;
-    text-align: center;
-    font-size: 16px;
-}
-.sort-icon {
-    display: flex;
-    height: 100%;
-    width: 30%;
-    justify-content: space-around;
-    align-items: center;
-}
-header .categories{
-    display: flex;
-    justify-content: center;
-    flex-wrap: wrap;
-}
-.categories button {
-    border-bottom: 2px solid transparent;
-    padding: 4px 0;
-    transition: 0.7s ease-out;
-    color: white;
-    margin: 10px 20px;
-    text-decoration: none;
-}
-.categories button:hover, .active {
-    color: #072C44;
-    border-bottom: 2px solid #072C44;
-}
+<<<<<<< HEAD
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    /*solo para mostrar contraste*/
+    background: red;
```

Comment:
A ver - creo decidimos verde si?
Como:
`background: green;`

Reply:
Ok thx for ur feedback

Reply:
en verdad era verde